Modules/_hashopenssl.c

```diff
@@ -31,15 +30,22 @@
 #define HASH_OBJ_CONSTRUCTOR 0
 #endif
 
-/* Minimum OpenSSL version needed to support sha224 and higher. */
```

Comment:
This was removed here: bc26646 essentially for dropping compatibility with older than 0.9.8 openssl versions.